packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -32,15 +34,6 @@ export class ElectronActions {
     this.electron.browserWindow = window
   }
 
-  hideBrowserWindow () {
-    // this.electron.browserWindow?.hide()
-    // if (this.isMac) {
-    //   this.ctx.electronApp?.dock.hide()
-    // } else {
-    //   this.electron.browserWindow?.setSkipTaskbar(true)
-    // }
-  }
-
```

Comment:
removed this and other related dead code... it was cropping up in searches and things like this should be recorded in tasks